lib/manager/regex/index.ts

```diff
@@ -43,15 +43,27 @@ function createDependency(
 ): PackageDependency {
   const dependency = dep || {};
   const { groups } = matchResult;
+
+  function updateDependency(field: string, value: string): void {
+    switch (field) {
+      case 'registryUrl':
+        // check if URL is valid and pack inside an array
+        if (url.parse(value)) {
+          dependency.registryUrls = [value];
+        }
```

Comment:
```suggestion
        } else {
          logger.warn({ value }, 'Invalid regex manager registryUrl');
        }
```

I think it would be good to warn?